lib/resources/Account.js

```diff
@@ -2,12 +2,40 @@
 
 var StripeResource = require('../StripeResource');
 var stripeMethod = StripeResource.method;
+var utils = require('../utils');
 
 module.exports = StripeResource.extend({
-  path: 'account',
+  // Since path can either be `account` or `accounts`, support both through stripeMethod path
```

Comment:
Nope, `/v1/account` is (in theory) 100% identical to `/v1/accounts/your_acct_id`. Essentially the latter acts as a convenience method to the former with a `Stripe-Account: your_acct_id` header.